client/reader/recommendations/posts/index.jsx

```diff
@@ -0,0 +1,26 @@
+var React = require( 'react' );
```

Comment:
Thanks @mtias! I can look into adjusting the folders to match the convention. @blowery, do we have a preference on one vs the other? I like the /reader/recommendations/posts URL so i'm leaning towards the posts.jsx route.